dataprofiler/profilers/graph_profiler.py

```diff
@@ -391,6 +393,11 @@ def _get_continuous_distribution(
             st.lognorm,
             st.gamma,
         ]
+
```

Comment:
I think this PR was updated from the wrong branch. 

We should fix so this only handles the num_quantiles and doesn't include the extra.

Reply:
pretty sure we just resolved the wrong branch issue